lib/foreman/model.rb

```diff
@@ -1,2 +0,0 @@
-module Foreman::Model
-end
```

Comment:
I think this is still being used at [compute resources](https://github.com/theforeman/foreman/blob/develop/app/models/compute_resources/foreman/model/vmware.rb#L4), we should probably keep it or change the definition there to
```ruby
module Foreman
  module Mode
  ...
```